src/GraphQL/Instrumentation/ApolloTrace.cs

```diff
@@ -13,7 +13,9 @@ public class ApolloTrace
         public ApolloTrace(DateTime start, double durationMs)
         {
             StartTime = start.ToUniversalTime();
-            EndTime = StartTime.AddMilliseconds(durationMs);
+            // https://github.com/dotnet/runtime/pull/73198 fixes AddMilliseconds, but it is only available NET7+
+            // Use AddTicks instead to fix it for all frameworks
+            EndTime = StartTime.AddTicks((long)(durationMs * TimeSpan.TicksPerMillisecond));
```

Comment:
👍

Reply:
This should be patched in a separate PR.  While perhaps not likely to matter, it can/will cause different behavior in the GraphQL runtime.

Reply:
- #3384

Reply:
Done